src/main/kotlin/sliv/tool/importer/view/ChoosingDirectoryView.kt

```diff
@@ -30,8 +37,29 @@ class ChoosingDirectoryView : Fragment() {
         }
 
         path.onChange {
-            rootTree = ProjectParser.createTreeWithFiles(it, rootTree)
+            project.set(parseDirectory(it.toString()))
+            rootTree = ProjectParser.createTreeWithFiles(project.value, rootTree)
+        }
+
+        treeview(rootTree) {
+            visibleWhen { path.isNotEmpty }
+        }
+
+        vbox {
+            padding = Insets(0.0, 0.0, 0.0, 10.0)
+            button("Import") {
+                this.isDisable = true
```

Comment:
this is neccessary?

Reply:
Probably yes. I think it would be better if the button is disabled when the project has not yet been selected